subprojects/language-native/src/integTest/groovy/org/gradle/language/nativeplatform/ParallelNativePluginsIntegrationTest.groovy

```diff
@@ -41,6 +43,7 @@ class ParallelNativePluginsIntegrationTest extends AbstractInstalledToolChainInt
     }
 
     @Requires(TestPrecondition.OBJECTIVE_C_SUPPORT)
```

Comment:
I reverted and used `TestPrecondition` as an enum to filter the right tool chain.